### PR TITLE
new icon: pandas (original, original-wordmark)

### PR DESCRIPTION
**Pull request by kimjiwook0129**
new icon: pandas (original, original-wordmark)

### CODE REVIEW

**Comment by theblobscp** [comment]
Why is numpy also in this branch history

**Comment by kimjiwook0129** [comment]
> Why is numpy also in this branch history

The branch history includes numpy files since I've opened those files to benchmark pandas files (to not make any mistake).
In such benchmark process, I've copied few lines to compare and I think that should have included in the history.
But nothing's been edited on 'icons/numpy/*'

Sorry for causing confusion :(
Is there anything I can do to remove redundancy on the branch history?

File: icons/pandas/pandas-original.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...h 12.25033 v 25.436342 h -12.25033 z" id="rect8" /><path style="color:#000000;fill:#130754;stroke-width:0.508523;-inkscape-stroke:none" d="m 48.697029,67.427074 h 12.25033 v 25.436345 h -12.25033 z" id="rect10" /><path style="color:#000000;fill:#ffca00;stroke-width:0.508523;-inkscape-stroke:none" d="m 48.697029,48.037076 h 12.25033 v 12.001153 h -12.25033 z" id="rect12" /> <path style="color:#000000;fill:#130754;stroke-width:0.508523;-inkscape-stroke:none" d="M 29.017171,36.086775 H 41.267502 V 120.63896 H 29.017171 Z" id="rect14" /> <path style="color:#000000;fill:#130754;stroke-width:0.508523;-inkscape-stroke:none" d="M 67.970068,88.413838 H 80.220394 V 113.85018 H 67.970068 Z" id="rect16" /> <path style="color:#000000;fill:#130754;stroke-width:0.508523;-inkscape-stroke:none" d="M 67.970068,36.117285 H 80.220394 V 61.553629 H 67.970068 Z" id="rect18" /> <path style="color:#000000;fill:#e70488;stroke-width:0.508523;-inkscape-stroke:none" d="M 67.970068,68.983156 H 80.220394 V 80.984312 H 67.970068 Z" id="rect20" /> <path style="color:#000000;fill:#130754;stroke-width:0.508523;-inkscape-stroke:none" d="M 87.238019,8.5502295 H 99.488347 V 93.10242 H 87.238019 Z" id="rect22" /></svg>
```
Comment (Panquesito7) [inline icons/pandas/pandas-original.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path style="color:#000;fill:#130754;stroke-width:.508523;-inkscape-stroke:none" d="M48.697 15.176h12.25v25.437h-12.25zm0 52.251h12.25v25.436h-12.25z"/><path style="color:#000;fill:#ffca00;stroke-width:.508523;-inkscape-stroke:none" d="M48.697 48.037h12.25v12.001h-12.25z"/><path style="color:#000;fill:#130754;stroke-width:.508523;-inkscape-stroke:none" d="M29.017 36.087h12.25v84.552h-12.25zM67.97 88.414h12.25v25.436H67.97zm0-52.297h12.25v25.437H67.97z"/><path style="color:#000;fill:#e70488;stroke-width:.508523;-inkscape-stroke:none" d="M67.97 68.983h12.25v12.001H67.97z"/><path style="color:#000;fill:#130754;stroke-width:.508523;-inkscape-stroke:none" d="M87.238 8.55h12.25v84.552h-12.25z"/></svg><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path style="color:#000;fill:#130754;stroke-width:.508523;-inkscape-stroke:none" d="M48.697 15.176h12.25v25.437h-12.25zm0 52.251h12.25v25.436h-12.25z"/><path style="color:#000;fill:#ffca00;stroke-width:.508523;-inkscape-stroke:none" d="M48.697 48.037h12.25v12.001h-12.25z"/><path style="color:#000;fill:#130754;stroke-width:.508523;-inkscape-stroke:none" d="M29.017 36.087h12.25v84.552h-12.25zM67.97 88.414h12.25v25.436H67.97zm0-52.297h12.25v25.437H67.97z"/><path style="color:#000;fill:#e70488;stroke-width:.508523;-inkscape-stroke:none" d="M67.97 68.983h12.25v12.001H67.97z"/><path style="color:#000;fill:#130754;stroke-width:.508523;-inkscape-stroke:none" d="M87.238 8.55h12.25v84.552h-12.25z"/></svg><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path style="color:#000;fill:#130754;stroke-width:.508523;-inkscape-stroke:none" d="M48.697 15.176h12.25v25.437h-12.25zm0 52.251h12.25v25.436h-12.25z"/><path style="color:#000;fill:#ffca00;stroke-width:.508523;-inkscape-stroke:none" d="M48.697 48.037h12.25v12.001h-12.25z"/><path style="color:#000;fill:#130754;stroke-width:.508523;-inkscape-stroke:none" d="M29.017 36.087h12.25v84.552h-12.25zM67.97 88.414h12.25v25.436H67.97zm0-52.297h12.25v25.437H67.97z"/><path style="color:#000;fill:#e70488;stroke-width:.508523;-inkscape-stroke:none" d="M67.97 68.983h12.25v12.001H67.97z"/><path style="color:#000;fill:#130754;stroke-width:.508523;-inkscape-stroke:none" d="M87.238 8.55h12.25v84.552h-12.25z"/></svg>

```

File: icons/pandas/pandas-original-wordmark.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...3.94193,16.057216 h 7.94743 v 16.501877 h -7.94743 z" id="rect131" /> <path style="color:#000000;fill:#130754;stroke-width:0.329904;-inkscape-stroke:none" d="m 53.94193,49.955009 h 7.94743 v 16.501879 h -7.94743 z" id="rect133" /> <path style="color:#000000;fill:#ffca00;stroke-width:0.329904;-inkscape-stroke:none" d="m 53.94193,37.379009 h 7.94743 v 7.785771 h -7.94743 z" id="rect135" /> <path style="color:#000000;fill:#130754;stroke-width:0.329904;-inkscape-stroke:none" d="m 41.17459,29.626237 h 7.94743 v 54.853395 h -7.94743 z" id="rect137" /> <path style="color:#000000;fill:#130754;stroke-width:0.329904;-inkscape-stroke:none" d="m 66.44536,63.570219 h 7.94742 v 16.501867 h -7.94742 z" id="rect139" /> <path style="color:#000000;fill:#130754;stroke-width:0.329904;-inkscape-stroke:none" d="m 66.44536,29.64603 h 7.94742 v 16.501868 h -7.94742 z" id="rect141" /> <path style="color:#000000;fill:#e70488;stroke-width:0.329904;-inkscape-stroke:none" d="m 66.44536,50.964524 h 7.94742 v 7.785769 h -7.94742 z" id="rect143" /> <path style="color:#000000;fill:#130754;stroke-width:0.329904;-inkscape-stroke:none" d="m 78.94548,11.761847 h 7.94741 v 54.853394 h -7.94741 z" id="rect145" /> </svg>
```
Comment (Panquesito7) [inline icons/pandas/pandas-original-wordmark.svg:1]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path style="color:#000;fill:#130754;stroke-width:.217695;-inkscape-stroke:none" d="M16.25 112.609c-2.238 0-4.513-.925-5.941-2.739v9.796H5.833v-24.18h4.049l.159 2.651c1.389-2.044 3.857-3.124 6.21-3.121 4.511 0 7.838 3.779 7.838 8.795 0 5.015-3.329 8.801-7.84 8.799zm-1.387-14.27c-2.613 0-4.746 2.044-4.746 5.477 0 3.433 2.12 5.477 4.744 5.477 2.623 0 4.739-2.044 4.739-5.477 0-3.433-2.116-5.477-4.737-5.477zm26.095 13.811-.155-2.66c-1.388 2.043-3.857 3.123-6.208 3.123-4.513 0-7.837-3.781-7.837-8.795 0-5.013 3.317-8.795 7.837-8.792 2.35 0 4.82 1.088 6.208 3.123l.155-2.66h4.05v16.663zm-4.974-13.81c-2.613 0-4.744 2.044-4.744 5.477 0 3.433 2.12 5.477 4.744 5.477 2.623 0 4.743-2.044 4.743-5.475 0-3.43-2.12-5.466-4.743-5.479zm24.678 13.814v-9.228c0-3.24-1.12-4.32-3.163-4.32-2.12 0-4.435 1.927-4.437 4.241v9.296H48.59V95.489h4.1l.191 3.087c1.12-2.122 3.627-3.548 6.172-3.548 4.434 0 6.095 3.087 6.095 7.214v9.912zm20.877.002-.155-2.662c-1.387 2.044-3.855 3.123-6.209 3.123-4.512 0-7.837-3.78-7.837-8.794 0-5.014 3.318-8.793 7.837-8.793 2.236 0 4.513.927 5.94 2.74v-8.755h4.473v23.141zm-4.972-13.802c-2.624 0-4.746 2.044-4.746 5.477 0 3.433 2.12 5.475 4.743 5.478 2.624 0 4.744-2.045 4.744-5.478s-2.12-5.477-4.741-5.477zm25.833 13.804-.154-2.66c-1.389 2.044-3.858 3.124-6.209 3.122-4.513 0-7.837-3.78-7.837-8.793 0-5.014 3.318-8.795 7.837-8.793 2.351 0 4.82 1.089 6.209 3.124l.155-2.66h4.06v16.66zm-4.974-13.804c-2.623 0-4.743 2.044-4.743 5.477 0 3.433 2.12 5.478 4.743 5.478s4.744-2.045 4.744-5.478-2.12-5.477-4.744-5.477zm18.938 14.346a13.094 13.094 0 0 1-6.673-1.89l.732-3.095c1.35.81 3.394 1.853 5.862 1.853 1.774 0 3.009-.54 3.009-1.93 0-1.194-1.272-1.619-3.549-2.159-4.086-.888-5.553-3.047-5.551-5.4 0-2.613 2.044-5.09 6.557-5.09 2.739 0 5.129 1.197 5.66 1.504l-.725 2.95a9.692 9.692 0 0 0-4.82-1.39c-1.813 0-2.701.617-2.701 1.62 0 1.117 1.158 1.62 2.932 2.044 4.395.926 6.17 2.97 6.17 5.208 0 3.579-2.624 5.775-6.903 5.775z"/><path style="color:#000;fill:#130754;stroke-width:.329904;-inkscape-stroke:none" d="M53.942 16.057h7.947V32.56h-7.947zm0 33.898h7.947v16.502h-7.947z"/><path style="color:#000;fill:#ffca00;stroke-width:.329904;-inkscape-stroke:none" d="M53.942 37.379h7.947v7.786h-7.947z"/><path style="color:#000;fill:#130754;stroke-width:.329904;-inkscape-stroke:none" d="M41.175 29.626h7.947V84.48h-7.947zm25.27 33.944h7.948v16.502h-7.948zm0-33.924h7.948v16.502h-7.948z"/><path style="color:#000;fill:#e70488;stroke-width:.329904;-inkscape-stroke:none" d="M66.445 50.965h7.948v7.785h-7.948z"/><path style="color:#000;fill:#130754;stroke-width:.329904;-inkscape-stroke:none" d="M78.945 11.762h7.948v54.853h-7.948z"/></svg>

```

File: devicon.json [hunk 3]
```diff
@@ -3052,6 +3052,30 @@
...
+                "original",
+                "original-wordmark"
+            ]
+        },
+        "color": "#130754",
+        "aliases": [
+            {
+                "base": "original",
+                "alias": "plain"
+            }
```
Comment (Panquesito7) [inline devicon.json:3076]:
```suggestion
            },
            {
                "base": "original-wordmark",
                "alias": "plain-wordmark"
            }
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by kimjiwook0129** [comment]
Requested changes are now reflected.

**Comment by kimjiwook0129** [comment]
It seems all checks have failed after the changes on the SVGs due to their qualities. Shall I try resizing the them (maybe using svgminifyer)?

File: icons/pandas/pandas-original.svg [hunk 4]
```diff
@@ -0,0 +1 @@
...z"/><path style="color:#000;fill:#130754;stroke-width:.508523;-inkscape-stroke:none" d="M29.017 36.087h12.25v84.552h-12.25zM67.97 88.414h12.25v25.436H67.97zm0-52.297h12.25v25.437H67.97z"/><path style="color:#000;fill:#e70488;stroke-width:.508523;-inkscape-stroke:none" d="M67.97 68.983h12.25v12.001H67.97z"/><path style="color:#000;fill:#130754;stroke-width:.508523;-inkscape-stroke:none" d="M87.238 8.55h12.25v84.552h-12.25z"/></svg><svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path style="color:#000;fill:#130754;stroke-width:.508523;-inkscape-stroke:none" d="M48.697 15.176h12.25v25.437h-12.25zm0 52.251h12.25v25.436h-12.25z"/><path style="color:#000;fill:#ffca00;stroke-width:.508523;-inkscape-stroke:none" d="M48.697 48.037h12.25v12.001h-12.25z"/><path style="color:#000;fill:#130754;stroke-width:.508523;-inkscape-stroke:none" d="M29.017 36.087h12.25v84.552h-12.25zM67.97 88.414h12.25v25.436H67.97zm0-52.297h12.25v25.437H67.97z"/><path style="color:#000;fill:#e70488;stroke-width:.508523;-inkscape-stroke:none" d="M67.97 68.983h12.25v12.001H67.97z"/><path style="color:#000;fill:#130754;stroke-width:.508523;-inkscape-stroke:none" d="M87.238 8.55h12.25v84.552h-12.25z"/></svg>
```
Comment (Panquesito7) [inline icons/pandas/pandas-original.svg:1]: This suggestion should fix the CI failure.

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><path style="color:#000;fill:#130754;stroke-width:.508523;-inkscape-stroke:none" d="M48.697 15.176h12.25v25.437h-12.25zm0 52.251h12.25v25.436h-12.25z"/><path style="color:#000;fill:#ffca00;stroke-width:.508523;-inkscape-stroke:none" d="M48.697 48.037h12.25v12.001h-12.25z"/><path style="color:#000;fill:#130754;stroke-width:.508523;-inkscape-stroke:none" d="M29.017 36.087h12.25v84.552h-12.25zM67.97 88.414h12.25v25.436H67.97zm0-52.297h12.25v25.437H67.97z"/><path style="color:#000;fill:#e70488;stroke-width:.508523;-inkscape-stroke:none" d="M67.97 68.983h12.25v12.001H67.97z"/><path style="color:#000;fill:#130754;stroke-width:.508523;-inkscape-stroke:none" d="M87.238 8.55h12.25v84.552h-12.25z"/></svg>

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by Thomas-Boi** [comment]
For some reason GitHub is not allowing us to view the logs of the workflow. We'll have to wait and see what issue is later

**Comment by Panquesito7** [comment]
Issues with the CI have been fixed now.

**Review by Panquesito7 — APPROVED** [review]
LGTM. Amazing work, @kimjiwook0129; thank you for your contribution! 😄👍🎉

**Comment by kimjiwook0129** [comment]
Thank you!

**Comment by Thomas-Boi** [comment]
Hi @kimjiwook0129,

The icons look good to me. However, we are currently pausing accepting new icons until the next release. This is due to an upgrade to our peek and check-svg bot that we need to implement before checking further icon.

This PR will be left open until the next release. Once that's done, we will re-run peek-bot and merge it then 👍

**Comment by kimjiwook0129** [comment]
Sounds good, thank you!

**Review by Thomas-Boi — APPROVED** [review]
The icons look good both as SVG and icons! Thank you @kimjiwook0129

**Pull request merged**